iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -235,6 +236,10 @@ public static String[] selectedColumns(Configuration conf) {
     return readColumns.split(conf.get(serdeConstants.COLUMN_NAME_DELIMITER, String.valueOf(SerDeUtils.COMMA)));
   }
 
+  public static boolean fetchVirtualColumns(Configuration conf) {
```

Comment:
Should we store this on table level, instead of globally?
If there is a query reading an iceberg table, and deleting from another table, then we do not need to read the virtual columns for the table we are just reading.